redisinsight/ui/src/utils/transformers/transformRdiPipeline.ts

```diff
@@ -9,15 +27,40 @@ export const pipelineToYaml = (pipeline: IPipelineJSON) => ({
   }))
 })
 
-export const pipelineToJson = ({ config, jobs }: IPipeline): IPipelineJSON => <IPipelineJSON>({
-  config: yaml.load(config) || {},
-  jobs: jobs.reduce<{ [key: string]: unknown }>((acc, job) => {
-    acc[job.name] = yaml.load(job.value)
+export const pipelineToJson = ({ config, jobs }: IPipeline, onError: (errors: IYamlFormatError[]) => void) => {
+  const result: IPipelineJSON = {
+    config: {},
+    jobs: []
+  }
+  const errors = []
+
+  try {
+    result.config = yaml.load(config) || {}
+  } catch (e) {
+    if (e instanceof YAMLException) {
+      errors.push({ filename: 'config', msg: e.reason })
+    }
+  }
```

Comment:
could you please use here yamlToJson function

Reply:
Good point. Thank you.